src/DataProtection/DataProtection/src/Managed/ManagedAuthenticatedEncryptor.cs

```diff
@@ -144,20 +147,28 @@ private SymmetricAlgorithm CreateSymmetricAlgorithm()
 
         retVal.Mode = CipherMode.CBC;
         retVal.Padding = PaddingMode.PKCS7;
+
         return retVal;
     }
 
-    private KeyedHashAlgorithm CreateValidationAlgorithm(byte[] key)
+    private KeyedHashAlgorithm CreateValidationAlgorithm()
```

Comment:
Nit: Remove? The other method uses a default value so should work for all current callsites.

Reply:
ye, removed